pkg/document/generate.go

```diff
@@ -26,7 +26,7 @@ func getOutputFile(chartDirectory string, dryRun bool) (*os.File, error) {
 	return f, err
 }
 
-func PrintDocumentation(chartDocumentationInfo helm.ChartDocumentationInfo, chartSearchRoot string, templateFiles []string, dryRun bool, helmDocsVersion string, badgeStyle string, dependencyValues []DependencyValues) {
+func PrintDocumentation(chartDocumentationInfo helm.ChartDocumentationInfo, chartSearchRoot string, templateFiles []string, dryRun bool, helmDocsVersion string, badgeStyle string, dependencyValues []DependencyValues, skipVersionFooter bool) {
```

Comment:
I noticed this parameter list is getting long, I can create a struct to group similar parameters together either in this change or a follow up change if preferred.

For example, a PrintDocumentationParams for all params or a TemplateOpts struct that groups together settings like `badgeStyle`, `helmDocsVersion`, and `skipVersionFooter` that control the default template's behavior.